packages/icons-vue/examples/vue-cli/yarn.lock

```diff
@@ -1,7738 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
is this file deletion intentional?

Reply:
Not entirely clear when that file was touched. Thanks noticing it though.